src/main/kotlin/app/batch/Config.kt

```diff
@@ -20,11 +20,11 @@ object Config {
             setInt("hbase.client.keyvalue.maxsize", getEnv("K2HB_HBASE_KEYVALUE_MAXSIZE")?.toIntOrNull() ?: 0)
         }
 
-        val dataTable = getEnv("K2HB_HBASE_DATA_TABLE") ?: "k2hb:ingest"
-        val dataFamily = getEnv("K2HB_HBASE_DATA_FAMILY") ?: "topic"
-        val topicTable = getEnv("K2HB_HBASE_TOPIC_TABLE") ?: "k2hb:ingest-topic"
-        val topicFamily = getEnv("K2HB_HBASE_TOPIC_FAMILY") ?: "c"
-        val topicQualifier = getEnv("K2HB_HBASE_TOPIC_QUALIFIER") ?: "msg"
+        val dataTable = data_table
```

Comment:
Snake case in java?

Reply:
yeah I wondered... It's matching the case from the logger stuff. I can change it.